pkgs/applications/gis/qgis/unwrapped.nix

```diff
@@ -141,6 +142,11 @@ in mkDerivation rec {
       pyQt5PackageDir = "${py.pkgs.pyqt5}/${py.pkgs.python.sitePackages}";
       qsciPackageDir = "${py.pkgs.qscintilla-qt5}/${py.pkgs.python.sitePackages}";
     })
+    (fetchpatch {
```

Comment:
If possible, I would try to use upstream [patch](https://github.com/qgis/QGIS/commit/32f5418fc4f7bb2ee986dee1824ff2989c113a94) for exiv 0.28.

Reply:
Done.